File: gdscript/src/main/java/gdscript/psi/GdGetDecl.java
package gdscript.psi;

import org.jetbrains.annotations.*;
import com.intellij.psi.PsiElement;

public interface GdGetDecl extends PsiElement {

  @Nullable
  GdEndStmt getEndStmt();

  @Nullable
  GdGetMethodIdRef getGetMethodIdNm();

  @Nullable
  GdStmtOrSuite getStmtOrSuite();

}

File: gdscript/src/main/java/gdscript/psi/GdGetDecl.kt
package gdscript.psi;

import com.intellij.psi.PsiElement;
import org.jetbrains.annotations.Nullable;

public interface GdGetDecl extends PsiElement {

  @Nullable
  GdEndStmt getEndStmt();

  @Nullable
  GdGetMethodIdRef getGetMethodIdNm();

  @Nullable
  GdStmtOrSuite getStmtOrSuite();

}
